# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../gnome-sharp/enable-gtkhtml-3.14.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -ur gnome-sharp-2.16.0.orig/configure.in gnome-sharp-2.16.0/configure.in
--- gnome-sharp-2.16.0.orig/configure.in	2006-08-21 18:46:13.000000000 +0200
+++ gnome-sharp-2.16.0/configure.in	2007-08-28 13:49:02.000000000 +0200
@@ -195,6 +195,12 @@
 PKG_CHECK_MODULES(RSVG_DEPENDENCIES, librsvg-2.0 >= $RSVG_REQUIRED_VERSION, enable_rsvg=yes, enable_rsvg=no)
 
 if test "x$enable_gnome" = "xyes"; then
+GTKHTML_REQUIRED_VERSION=3.14.0
+PKG_CHECK_MODULES(GTKHTML_DEPENDENCIES, libgtkhtml-3.14 >= $GTKHTML_REQUIRED_VERSION, enable_gtkhtml=yes, enable_gtkhtml=no)
+if test "x$enable_gtkhtml" = "xyes"; then
+GTKHTML_VERSION=3.14
+GTKHTML_SOVERSION=19
+else
 GTKHTML_REQUIRED_VERSION=3.8.0
 PKG_CHECK_MODULES(GTKHTML_DEPENDENCIES, libgtkhtml-3.8 >= $GTKHTML_REQUIRED_VERSION, enable_gtkhtml=yes, enable_gtkhtml=no)
 if test "x$enable_gtkhtml" = "xyes"; then
@@ -225,6 +231,7 @@
 fi
 fi
 fi
+fi
 AC_SUBST(GTKHTML_VERSION)
 AC_SUBST(GTKHTML_SOVERSION)
 else
